charts/sysdig/templates/secrets.yaml

```diff
@@ -14,9 +14,9 @@ data:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "sysdig.fullname" . }}
+  name: {{ .name }}
   labels:
-{{ include "sysdig.labels" . | indent 4 }}
+{{ include "sysdig.labels" $ | indent 4 }}
```

Comment:
question: why is the dollar sign preferred?

Reply:
inside a range you change 'context' and '.' is the relative path to the current context.
with '$' you are specifying to pass all the values from the root

Reply:
interesting. good to know, thank you!